Gruntfile.js

```diff
@@ -16,13 +16,55 @@ module.exports = function(grunt) {
     },
     browserify: {
       standalone: {
-        src: [ 'src/<%= pkg.name %>.js' ],
+        src: 'src/<%= pkg.name %>.js',
         dest: 'dist/<%= pkg.name %>.js',
         options: {
           browserifyOptions: {
-            standalone: 'ContextParserHandlebars'
+            standalone: 'Handlebars'
```

Comment:
browserify version is for client-side use like <script src="">
to meet the goal of changing only the URL, we should override the original Handlebars, right?